administration/src/components/generation/EakGeneration.tsx

```diff
@@ -57,30 +57,57 @@ const FormColumn = styled.div`
   box-sizing: border-box;
 `;
 
+const PdfWrapper = styled.div`
+  position: absolute;
+  display: flex;
+  padding-top: 120px;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  bottom: 0;
+`;
 
 const EakGeneration = () => {
 
     const [cardCreationModels, setCardCreationModels] = useState([createEmptyCard()]);
+    const [pdfView, setPdfView] = useState(false);
 
     const addForm = () => setCardCreationModels([...cardCreationModels, createEmptyCard()]);
     const removeForm = (id: number) => setCardCreationModels([...cardCreationModels].filter( model => model.id != id));
     const reset = () => setCardCreationModels([createEmptyCard()]);
+    const switchView = () => setPdfView(!pdfView);
 
     const forms = cardCreationModels.map(ccm => <FormColumn key={ccm.id}><EakForm onRemove={() => removeForm(ccm.id)}/></FormColumn>); // Todo!
 
+    const view = pdfView ? (
```

Comment:
Whis probably breaks navigating back. e.g. what happens if you navigate back in the browser after generating the pdfs?

Reply:
True, it does. Ideas for fixing?

Reply:
This needs to be integrated as a separate view in the navigator